stories/s2l2a.stories.js

```diff
@@ -204,7 +218,7 @@ export const GetMapProcessingWithGeometry = () => {
       bbox: bbox4326,
       fromTime: new Date(Date.UTC(2018, 11 - 1, 22, 0, 0, 0)),
       toTime: new Date(Date.UTC(2018, 12 - 1, 22, 23, 59, 59)),
-      geometry: geometry,
+      geometry: geometryPolygon,
```

Comment:
`geometryPolygon` is not defined.

Reply:
Last moment changes, sorry... :) Thx, fixed.